views-pebble/src/main/java/io/micronaut/views/pebble/PebbleViewsRenderer.java

```diff
@@ -38,15 +42,18 @@
 @Requires(property = PebbleConfigurationProperties.ENABLED, notEquals = StringUtils.FALSE)
 @Requires(classes = PebbleEngine.class)
 public class PebbleViewsRenderer<T> implements ViewsRenderer<T> {
-    
+
     private final PebbleEngine engine;
+    private final HttpLocaleResolver httpLocaleResolver;
 
     /**
      * @param engine Pebble Engine
```

Comment:
it misses javadoc for `httpLocaleResolver` parameter

Reply:
:disappointed:
Fixed